examples/generate.sh

```diff
@@ -102,6 +104,9 @@ echo "Generated ${METAL3PLANE_GENERATED_FILE}"
 kustomize build "${SOURCE_DIR}/machinedeployment" | envsubst >> "${MACHINEDEPLOYMENT_GENERATED_FILE}"
 echo "Generated ${MACHINEDEPLOYMENT_GENERATED_FILE}"
 
+# Get Cert-manager provider components file
+curl -L -o "${COMPONENTS_CERT_MANAGER_GENERATED_FILE}" https://github.com/jetstack/cert-manager/releases/download/v0.12.0/cert-manager.yaml
```

Comment:
v0.13 is already available. Is there any particular reason not to use it? I saw CAPI still using 0.11 though and they have an open issue to upgrade to v0.13

Reply:
The main reason is that 0.11 was the version where cert-manager changed into v1alpha2 version. Upgrading from 0.11 to 0.12 there is some additional work required. 
https://cert-manager.io/docs/installation/upgrading/upgrading-0.11-0.12/

I would say let's try to make everything work with 0.11 for now?

Reply:
I'll upgrade to 0.13